src/gui/updater/ocupdater.cpp

```diff
@@ -184,6 +184,81 @@ void OCUpdater::setDownloadState(DownloadState state)
     }
 }
 
+namespace {
```

Comment:
minor: we usually just declare the functions `static` - here you do a mixture of both

Reply:
A mixture of what?

Reply:
Of an anonymous namespace and `static`. Both mean internal linkage for functions.

Reply:
Ah, yes. I usually do the anonymous namespace and copied the static from the qt source.